index.html

```diff
@@ -127,7 +127,6 @@
             'error', loadErrHandler, true /* capture phase type of listener */);
     </script>
     <script><!--#include virtual="/config.js" --></script><!-- adapt to your needs, i.e. set hosts and bosh path -->
-    <script src="static/utils.js?v=1"></script>
```

Comment:
Yes, but the difference is in __1kB__

There is `do_external_connect.js` without import statements (minified) - __1173 bytes__:
<img width="572" alt="screen shot 2017-04-13 at 3 57 48 pm" src="https://cloud.githubusercontent.com/assets/5806075/25005746/cc1585b4-2062-11e7-8818-2341072c1139.png">

And the same file importing functions from `configUtils.js` - __2027 bytes__:
<img width="566" alt="screen shot 2017-04-13 at 4 01 40 pm" src="https://cloud.githubusercontent.com/assets/5806075/25005814/12a0737c-2063-11e7-94ea-7f157bd05a5d.png">

As an advantage of this approach we've got rid of these functions in global scope and as a drawback we download additional __1KB__ of JS. I think it's better that storing them just in global scope and we can neglect especially because these changes `do_external_connect` was being loaded in non-minified version (__2388 bytes__):
<img width="566" alt="screen shot 2017-04-13 at 4 09 40 pm" src="https://cloud.githubusercontent.com/assets/5806075/25005951/aaefeda6-2063-11e7-8b17-fd52ed5a824b.png">